src/components/Modal/BaseModal.js

```diff
@@ -8,86 +8,101 @@ import themeColors from '../../styles/themes/default';
 import {propTypes, defaultProps} from './ModalPropTypes';
 import getModalStyles from '../../styles/getModalStyles';
 
-const BaseModal = (props) => {
-    const subscribeToKeyEvents = () => {
-        KeyboardShortcut.subscribe('Escape', props.onClose, [], true);
-        KeyboardShortcut.subscribe('Enter', props.onSubmit, [], true);
+class BaseModal extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.subscribeToKeyEvents = this.subscribeToKeyEvents.bind(this);
+        this.unsubscribeFromKeyEvents = this.unsubscribeFromKeyEvents.bind(this);
+    }
+
+    componentWillUnmount() {
+        this.unsubscribeFromKeyEvents();
+    }
+
+    subscribeToKeyEvents = () => {
```

Comment:
Can we avoid using the class property syntax here? We have standardized on just using regular class methods in components like these and since you've bound these in the constructor there is even less of a reason to do this here.